### ISSUE TITLE
Request for info: Are compression ratios expected to be the same for "all genomes compressed at once" vs. "genomes added one by one"?

### ISSUE THREAD

**Issue by karel-brinda** [kowallus/mbgc#12]

**Comment by kowallus** [comment]
No, they aren't. It might be the case in _max mode_, but there is no such guarantee.

**Comment by karel-brinda** [comment]
Ok; understand. This is more for understanding in the max mode, what's the penalty for future extensions of archived datasets, or whether it's better to recompress them from scratch (eg when merging two batches in phylogenetic compression).

**Comment by kowallus** [comment]
Appending might result in a slightly better ratio.
